data/Templates/Hl7v2/Resource/_Provenance.liquid

```diff
@@ -1,8 +1,19 @@
+{% assign generationInstant = "{{ "" | now }}" -%}
+{% assign templateVersion = "3.5.1" -%}
+{% assign rootTemplate = Root_Template -%}
```

Comment:
We can move them in `{% if Root_Template -%}` so we do not need to generate them for every Provenance. Also it looks that we can omit some variable assignment here. Maybe `templateVersion` would be enough.

Reply:
I've removed `rootTemplate` and `templateUrl`, and move other variables into `IF` block.